gemrb/GemRB.cfg.sample.in

```diff
@@ -29,6 +29,7 @@
 #                                                   #
 #  bg1       Baldur's Gate                          #
 #  bg2       Baldur's Gate 2 : SoA or ToB           #
+#  bg2ee     Baldur's Gate 2 : EE (experimental!)   #
```

Comment:
You didn't add any GameType detection files, but I think it's fine and should stay that way for a while to not give false hope.

Reply:
I'd like to skip that for now since I don't know exactly what's qualifying here.

Reply:
WeiDU has a list we can reuse later.